src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/RecordingOptionsBuilderFactory.java

```diff
@@ -45,7 +45,9 @@
 import org.openjdk.jmc.flightrecorder.configuration.recording.RecordingOptionsBuilder;
 import org.openjdk.jmc.rjmx.services.jfr.IFlightRecorderService;
 
-interface RecordingOptionsBuilderFactory {
+// FIXME this should be in a more general package, not commands/internal. This is also used in
```

Comment:
Does this comment also apply to EventOptionsBuilder?

Reply:
Yea, it does. Since essentially all of the commands are now deprecated, I figured it would make sense until they can be fully removed and then the RecordingOptionsBuilderFactory and EventOptionsBuilder can be moved somewhere close to the HTTP Handlers.

Reply:
Ah gotcha. Makes sense. Thanks